musit-service/src/main/scala/no/uio/musit/healthcheck/HealthCheckDao.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ *  MUSIT is a museum database to archive natural and cultural history data.
+ *  Copyright (C) 2016  MUSIT Norway, part of www.uio.no (University of Oslo)
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License,
+ *  or any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package no.uio.musit.healthcheck
+
+import com.google.inject.{Inject, Singleton}
+import play.api.db.slick.{DatabaseConfigProvider, HasDatabaseConfigProvider}
+import slick.driver.JdbcProfile
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class HealthCheckDao @Inject() (
+    val dbConfigProvider: DatabaseConfigProvider
+)(
+    implicit
+    ec: ExecutionContext
+) extends HasDatabaseConfigProvider[JdbcProfile] with HealthCheck {
+
+  import driver.api._
+
+  val CheckName = "Database"
```

Comment:
Hmm, tja. Ut fra naming convention til scala er det rett av den å pirke på det. Hvis den blir flyttet til ett object eller package så bør den kategoriseres som en konstant og ikke trigge på det.

>Constants, Values, Variable and Methods
>Constant names should be in upper camel case. Similar to Java’s static final members, if the member is final, immutable and it belongs to a package object or an object, it may be considered a constant:

http://docs.scala-lang.org/style/naming-conventions.html